migrate.md

```diff
@@ -0,0 +1,111 @@
+
+1.  Determine which resource group you want to move the service instance to first.
+
+    See [Best practices for organizing resource in a resource group ![External link icon](../../icons/launch-glyph.svg "External link icon")](/docs/resources?topic=resources-bp_resourcegroups) for tips.
+
+    With a Premium plan, you can migrate multiple instances. However, the instances must all be migrated to the same resource group.
+
+1.  From the *Cloud Foundry Services* section of the {{site.data.keyword.Bluemix_notm}} resource list, click the migrate icon ![Migrate](images/migrate.svg) for the instance you want to migrate, and then click **Migrate** from the popup.
+
+1.  Click **Continue**, and then choose a resource group.
+
+    If you have not created a resource group, you can create one now. A **default** resource group is available. Take time to understand how the groups are used and create one if necessary. The resource group that you choose here *cannot* be changed later.
+
+1.  Click **Migrate**.
+
+    A message is displayed when the process is done. If you have other service instances to migrate, you can continue migrating other service instances, or click **Done**.
+
+1.  **Premium plan one-time step**: If the service instance you are migrating was created as part of a Premium plan, then you must inform the IBM support team that you are migrating a Premium plan instance. To do so, create a case from [IBM Cloud Support ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/unifiedsupport/supportcenter){: new_window}.
+
+    There are some additional steps that the IBM support team needs to take on your behalf. Add the following information to the case:
+
+    - Region in which the service instance is hosted, such as Dallas or Frankfurt
+    - Resource group name
+    - Resource group ID
+
+      If you don't know the ID, open the {{site.data.keyword.Bluemix_notm}} command line interface (CLI) tool and enter the following command:
+
+      ```bash
+      ibmcloud resource groups
+      ```
+      {: codeblock}
+
+      The response shows the ID of the resource group. For more information about the CLI command, see [Working with resources and resource groups ![External link icon](../../icons/launch-glyph.svg "External link icon")](https://cloud.ibm.com/docs/cli?topic=cloud-cli-ibmcloud_commands_resource#ibmcloud_commands_resource).
+
+      Give the support team up to 24 hours from the time you submit the case to complete the work they need to perform to support the Premium plan instance migration. There is no disruption of service during the migration process.
+
+      You only need to perform this step the first time you migrate a service instance for the Premium plan. When you click *Migrate* for other service instances that belong to the same plan, they will be migrated to the same resource group without requiring any involvement from the IBM support team.
```

Comment:
Consider using `for the first time that you migrate` for what I expect Acrolinx to flag.  Change `they will be migrated` to `they are migrated` to avoid future tense.